nodes/ati_ft_sensor_wrench_pub.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Title: ati_ft_sensor_wrench_pub.cpp
+ *
+ * Description: This file is part of ati_ft_sensor package. It publishes the
+ * wrench data from the ATI force torque sensor. It also publishes the wrench
+ * data to the faked_forces_controller based on a passed parameter, serving as a
+ * generic proxy for admittance control using PickNik's admittance controller.
+ * Follow the general template to replicate this for another FT sensor.
+ *
+ * Author: Emmanuel Akita
+ * Email:  efakita@utexas.edu
+ * Created: 4 Oct, 2023
+ *
+ */
+
+#include <AtiFTSensor.h>
+#include <chrono>
+
+// ROS includes
+#include "geometry_msgs/msg/wrench_stamped.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/float64_multi_array.hpp"
+
+class ATIWrenchPublisher : public rclcpp::Node {
+  // Constructor
+ public:
+  ATIWrenchPublisher() : Node("ati_wrench_publisher") {
+    initializeNode();
+
+    // Create a timer that calls the publish function every millisecond.
+    timer_ = this->create_wall_timer(
+        std::chrono::milliseconds(1),
+        std::bind(&ATIWrenchPublisher::publishSensorData, this));
+
+    if (!sensor_.initialize(ip_address_)) {
+      RCLCPP_ERROR(this->get_logger(), "Could not initialize the sensor.");
+    }
+  }
+
+  // Destructor
+  ~ATIWrenchPublisher() {
+    // Stop the timer.
+    timer_->cancel();
+  }
+
+ private:
+  rclcpp::TimerBase::SharedPtr timer_;
+  ati_ft_sensor::AtiFTSensor sensor_;
+  rclcpp::Publisher<geometry_msgs::msg::WrenchStamped>::SharedPtr pub_wrench_;
+  rclcpp::Publisher<std_msgs::msg::Float64MultiArray>::SharedPtr
+      pub_fake_controller_wrench_;
+  std::string frame_id_, ip_address_;
+  bool fake_controller_wrench_;
+
+  void initializeNode() {
+    // Get the required parameters
+    this->declare_parameter("frame_id", rclcpp::PARAMETER_STRING);
+    this->declare_parameter("ip_address", rclcpp::PARAMETER_STRING);
+    this->declare_parameter("pub_faked_wrench", rclcpp::PARAMETER_BOOL);
+
+    rclcpp::Parameter frame_id = this->get_parameter("frame_id");
```

Comment:
I think you can rewrite this on a more compact version.
frame_id_ = this->get_parameter("frame_id"); 
I dont think you need to load the param to get it again after, and then create a variable to store it as string.  (Same for the other params)